src/overlays/actors/ovl_En_Dns/z_en_dns.c

```diff
@@ -120,51 +109,45 @@ static InitChainEntry sInitChain[] = {
     ICHAIN_F32(targetArrowOffset, 30, ICHAIN_STOP),
 };
 
-typedef enum {
-    /* 0 */ ENDNS_ANIM_0,
-    /* 1 */ ENDNS_ANIM_1,
-    /* 2 */ ENDNS_ANIM_2
-} EnDnsAnimation;
-
 static AnimationMinimalInfo sAnimationInfo[] = {
     { &gBusinessScrubNervousIdleAnim, ANIMMODE_LOOP, 0.0f },
-    { &gBusinessScrubAnim_4404, ANIMMODE_ONCE, 0.0f },
+    { &gBusinessScrubLeaveBurrowAnim, ANIMMODE_ONCE, 0.0f },
     { &gBusinessScrubNervousTransitionAnim, ANIMMODE_ONCE, 0.0f },
 };
 
 void EnDns_Init(Actor* thisx, PlayState* play) {
     EnDns* this = (EnDns*)thisx;
 
-    if (this->actor.params < 0) {
+    if (EN_DNS_TYPE(this) < 0) {
         // "Function Error (Deku Salesman)"
-        osSyncPrintf(VT_FGCOL(RED) "引数エラー（売りナッツ）[ arg_data = %d ]" VT_RST "\n", this->actor.params);
+        osSyncPrintf(VT_FGCOL(RED) "引数エラー（売りナッツ）[ arg_data = %d ]" VT_RST "\n", EN_DNS_TYPE(this));
```

Comment:
the debug string shows the argument to the printf is really the raw `this->actor.params` (`arg_data` is its original name as many other similar printfs show), not the scrub's type